javascript/javascript1/week2/Weather-wear.js

```diff
@@ -0,0 +1,8 @@
+function weather(t) {
+    return t > 25 ? "It's beach time...!" 
+         : t > 15 && t <= 25 ? "It's time to wear a sweatshirt!"
```

Comment:
Tip: You can omit the second check (t <= 25) to reduce cognitive load (fewer conditions to consider).

```js
t > 25 ? "It's beach time...!"
: t > 15 ? "It's time to wear a sweatshirt!" // At this point you know that t<=25 is going to be true
// and so on
```

PS: Nice use of the ternary operator :sunglasses: